src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -39,7 +39,7 @@ class OrmExtension extends Nette\DI\CompilerExtension
 	 * @var array
 	 */
 	public $managerDefaults = array(
-		'metadataCache' => 'default',
+		'metadataCache' => null,
 		'queryCache' => 'default',
```

Comment:
It's because of [this line](https://github.com/doctrine/cache/blob/master/lib/Doctrine/Common/Cache/FileCache.php#L129).